janitor/functions.py

```diff
@@ -3309,3 +3309,74 @@ def take_a_if_even(x):
     """
     df = df.copy().join(df.apply(func, axis=1).rename(new_column_name))
     return df
+
+
+@pf.register_dataframe_method
+def flag_nulls(
+    df: pd.DataFrame,
+    column_name: str = "null_flag",
+    columns: Union[Iterable[str], str, None] = None,
+) -> pd.DataFrame:
+    """
+    Creates a new column to indicate whether you have null values in a given
+    row. If the columns parameter is not set, looks across the entire
+    DataFrame, otherwise will look only in the columns you set.
+
+    .. code-block:: python
```

Comment:
We recently started standardizing docstrings (h/t @loganthomas for kickstarting this). For pragmatic reasons (it’d be a lot of functions to cover), we haven’t put in toy examples in the docstrings, but I think in the long-run it’s a better idea to do what you’ve done. For now, we just have the “Functional” and “Method Chaining” examples, without inputs and outputs. 

Having written all that, let’s stick with what you’ve done here. I’ll make an issue to ask for the examples in the docstrings to be changed to minimal examples like you’ve provided, but nonetheless showcasing both the functional and method chaining styles.